devicon.json

```
@@ -4778,5 +4778,27 @@
                 "alias":"plain"
             }
         ]
+    },
+    {
+        "name": "pytest",
+        "tags": [
+            "python",
+            "framework",
+            "testing"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#386f9f",
```

I recommend more this color `#009fe3`
```suggestion
        "color": "#009fe3",
```

---

yes, i realized i picked the wrong one